shared/src/main/java/com/sap/psr/vulas/shared/json/model/ExemptionUnassessed.java

```diff
@@ -0,0 +1,138 @@
+/**
+ * This file is part of Eclipse Steady.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright (c) 2018 SAP SE or an SAP affiliate company. All rights reserved.
+ */
+package com.sap.psr.vulas.shared.json.model;
+
+import java.util.Map;
+
+import org.apache.commons.configuration.Configuration;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+public class ExemptionUnassessed implements IExemption {
+
+	private static final Log log = LogFactory.getLog(ExemptionUnassessed.class);
+
+	/**
+	 * Configuration setting <code>REP_EXCL_UNASS="vulas.report.exceptionExcludeUnassessed"</code>.
+	 */
+	public final static String CFG = "vulas.report.exceptionExcludeUnassessed";
```

Comment:
"vulas." should be removed to keep on using configurations without the vulas prefix as it was done until now